multiqc/modules/verifybamid/verifybamid.py

```diff
@@ -0,0 +1,238 @@
+                data[s_name][column] = d[column]
+
+        # create an ordered dictionary to preserve the order of columns
+        headers = OrderedDict()
+        # add each column and the title and description (taken from verifyBAMID website)
+        headers['#SEQ_ID']={
+            'title': 'Sample Name',
+            'description': 'Sample ID of the sequenced sample. Obtained from @RG header / SM tag in the BAM file',
+        }
+        headers['RG']={
+            'title': 'Read Group',
+            'description': 'ReadGroup ID of sequenced lane. For [outPrefix].selfSM and [outPrefix].bestSM, these values are "ALL',
+        }
+        headers['CHIP_ID'] = {
+            'title': 'Chip ID',
+            'description': 'ReadGroup ID of sequenced lane. For [outPrefix].selfSM and [outPrefix].bestSM, these values are "ALL',
+        }
+        headers['#SNPS']= {
+            'title': '#SNPS',
+            'description': '# of SNPs passing the criteria from the VCF file',
+        }
+        headers['#READS']= {
+            'title': '#Reads',
+            'description': 'Total # of reads loaded from the BAM file',
+        }
+        headers['AVG_DP']= {
+            'title': 'Average Depth',
+            'description': 'Average sequencing depth at the sites in the VCF file',
+        }
+        # specify the number of decimal points to display
+        headers['FREEMIX']= {
+            'title': 'FREEMIX',
+            'description': 'Sequence-only estimate of contamination (0-1 scale)',
+            'format': '{:,.5f}' 
+        }
+        headers['FREELK1']= {
+            'title': 'FREEELK1',
+            'description': 'Maximum log-likelihood of the sequence reads given estimated contamination under sequence-only method',
+        }
+        headers['FREELK0']= {
+            'title': 'FREELK0',
+            'description': 'Log-likelihood of the sequence reads given no contamination under sequence-only method',
+        }
+        headers['FREE_RH']= {
+            'title': 'FREE_RH',
+            'description': 'Estimated reference bias parameter Pr(refBase|HET) (when --free-refBias or --free-full is used)',
+        }
+        headers['FREE_RA']= {
+            'title': 'FREE_RA',
+            'description': 'Estimated reference bias parameter Pr(refBase|HOMALT) (when --free-refBias or --free-full is used)',
+        }
+        # specify the number of decimal points to display
+        headers['CHIPMIX']= {
+            'title': 'CHIPMIX',
+            'description': 'Sequence+array estimate of contamination (NA if the external genotype is unavailable) (0-1 scale)',
+            'format': '{:,.5f}' 
+        }
+        headers[ 'CHIPLK1']= {
+            'title': 'CHIPLK1',
+            'description': 'Maximum log-likelihood of the sequence reads given estimated contamination under sequence+array method (NA if the external genotypes are unavailable)',
```

Comment:
As above with read counts.

Could also have the _hide-if-all-NA_ logic on these columns too.